README.md

```diff
@@ -0,0 +1,82 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Pokemon Training Card Game
+
+> Browse throw 250 pokemon cards and get info like attacks, number of pokedex, value of the cards and pokemon type.
+
+<img src="https://github.com/AlexRS90/pokemon-cardgame/blob/mobile-app/ssHomePage.jpg" width="300" />
+<img src="https://github.com/AlexRS90/pokemon-cardgame/blob/mobile-app/ssDetailPage.jpg" width="300" />
+
+### Milestone 1:
+- Followed Gitflow
+- Created work-branch
+- Created a new react-app
+- Configured components
+- Installed Redux and dependencies
+  - Followed ducks pattern 🦆
+- Added Home Page component 🏠
+- Templated design followed 🎨
+  - Added styles 🧑‍🎨
+- Get data from store
+- Added Details component 🕵️
+- Showed info from pokemon selected
+- Followed template design
+- Added Navbar component 🌐
+  - Added back button ◀️
+  - Added drop down list 📃
+  - Filtered Pokemon by type 🔥 🌊 👁️ 🐉 🦾 🌱
+- Added dynamic routing for each single pokemon
+- Added unit test
+- Deployed web app
+- Recorded video presentation
+
+## Built With
+
+- React & Redux
+- Visual Studio Code
+- Linters
+- Deployed in Heroku
+
+## Live demo
+
+[PokemonTCG-SPA](https://pokemontcgcapstone.herokuapp.com/)
```

Comment:
[OPTIONAL] Good job with the video. Your English is really good and understandable, and the video itself gives good insight into the motivation to build this project and the code you wrote. 

Why not include it on the README file, this could be potentially decisive for a recruiter that might visit the GitHub of the project.

Reply:
Wow, thanks a lot @D4nielJ  🥳  it is so motivating! I'll add it in the README, thanks for all your suggestions!!
Happy Christmas 🎄  🎅